diff --git a/contrib/profile2_page.info b/contrib/profile2_page.info
index b57598e..3d55afe 100644
--- a/contrib/profile2_page.info
+++ b/contrib/profile2_page.info
@@ -2,7 +2,7 @@ name = Profile2 pages
 description = Adds separate pages for viewing and editing profiles.
 core = 7.x
 files[] = includes/translation.handler.profile2_page.inc
-dependencies[] = profile2; Information added by Drupal.org packaging script on 2018-09-08
+dependencies[] = profile2
 version = "7.x-1.5"
 core = "7.x"
 project = "profile2"
